pkg/diffs/diffs.go

```diff
@@ -51,13 +57,36 @@ func GetChangedCiopConfigs(masterConfig, prConfig config.CompoundCiopConfig, log
 			continue
 		}
 
-		if !equality.Semantic.DeepEqual(oldConfig, newConfig) {
+		withoutTestsOldConfig := cioperatorapi.ReleaseBuildConfiguration{}
+		deepcopy.Copy(&withoutTestsOldConfig, masterConfig[filename])
+		withoutTestsOldConfig.Tests = nil
+
+		withoutTestsNewConfig := cioperatorapi.ReleaseBuildConfiguration{}
+		deepcopy.Copy(&withoutTestsNewConfig, prConfig[filename])
+		withoutTestsNewConfig.Tests = nil
```

Comment:
We could avoid copying (and the external dep) if we saved tests, did the compare and put them back afterwards... Just a proposal for consideration, not a must-have.

```
oldTests := masterConfig[filename].Tests
masterConfig[filename].Tests = prConfig[filename].Tests // so that they are same
...compare...
masterConfig[filename].Tests = oldTests
```

Reply:
My intension here is actually to compare all the rest without the tests, since the tests are already saved in the original struct. My comparison holds a logic that if we found changes only in the `tests` field we should get the corresponding job is affected. 

If there are tests changes and some of the rest fields are changed also, we want to rehearse everything instead.

Reply:
Yeah, I know - I just suggested how how do the same without copying the whole configs around.

Reply:
We can always compare each field separately. I will check if this will be more efficient.

Reply:
I could use the address of the value here.  addressed